docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -62,7 +62,34 @@ There you can find links to upgrade notes for other versions too.
     - read the section about proxying the URL content subpaths via webserver domain [`docs/introduction/abstract-filesystem.md`](https://github.com/shopsys/shopsys/blob/master/docs/introduction/abstract-filesystem.md)
 - to be more descriptive about error caused by active TEST environment ([#701](https://github.com/shopsys/shopsys/pull/701))
     - modify `ErrorController::createUnableToResolveDomainResponse()` by these [changes](https://github.com/shopsys/shopsys/pull/701/files#diff-0b1aecbf82624ce474ca3cb8bd75811c).
-
+- start filtering products with elasticsearch ([#943](https://github.com/shopsys/shopsys/pull/943))
+    - replace Elasticsearch structure files in `src/Shopsys/ShopBundle/Resources/definition/products/` folder in your project with new ones from [GitHub](https://github.com/shopsys/project-base/blob/master/src/Shopsys/ShopBundle/Resources/definition/products/)
+    - replace class `Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacade` with the interface `Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacadeInterface` in
+        - `src/Shopsys/ShopBundle/Controller/Front/ProductController.php`
+        - `src/Shopsys/ShopBundle/Controller/Front/SearchController.php`
+    - add service definition for facade interface to your `services.yml` like `Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacadeInterface: '@Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainElasticFacade'`
```

Comment:
This seems little more readable to me (please check if the formatting is good :smile:)
```suggestion
    - add service definition for facade interface to your `services.yml` like 
        ```yaml
        Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacadeInterface: '@Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainElasticFacade'
        ```
```